common/shlibs

```diff
@@ -3745,3 +3745,6 @@ libigdgmm.so.11 intel-gmmlib-19.4.1_1
 libgtk-layer-shell.so.0 wf-shell-0.3_1
 librdkafka.so.1 librdkafka-1.3.0_1
 librdkafka++.so.1 librdkafka-1.3.0_1
+libxmpp-vala.so.0 dino-0.1.0_1
+libqlite.so.0 dino-0.1.0_1
+libdino.so.0 dino-0.1.0_1
```

Comment:
either drop the shlibs, or properly split off `libdino` and `libdino-devel`. We don't need to provide shlibs if nothing will depend on them, and unless the library and devel files are correctly split off, that won't be the case

Reply:
This is needed because if I delete these lines, I get:
```
=> dino-0.1.0_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libdino.so.0 <-> UNKNOWN PKG PLEASE FIX!
   SONAME: libxmpp-vala.so.0 <-> UNKNOWN PKG PLEASE FIX!
   SONAME: libqlite.so.0 <-> UNKNOWN PKG PLEASE FIX!
...
=> ERROR: dino-0.1.0_1: cannot guess required shlibs, aborting!
```

https://travis-ci.org/void-linux/void-packages/jobs/645756802?utm_medium=notification&utm_source=github_status

Reply:
ah, there's a selfdep, okay.